lightcurve/static/lightcurve/js/lightcurve.js

```diff
@@ -61,4 +73,19 @@ $(function() {
     window.location.reload();
     return false;
   });
+
+  $(document).on('click', '.edit-name', function(e) {
+    e.preventDefault();
```

Comment:
`return false` at the end of a jquery event handler is equivalent to `e.preventDefault()` and `e.stopPropagation()`.  This is a preferred convention.

Reply:
Done.